### DISCUSSION TITLE
Ability to disable spam settings on a per form basis

### DISCUSSION THREAD

**Discussion by terryupton** [solspace/craft-freeform#663] [Feature Requests]
As the title says really.
I would like the ability to disable spam settings on a per form basis. 

Context:
Client has several forms that require payment and booking and these go through stripe, so there is very limited chance these wold be spam submissions. As well as this client has a 5 step form that is a craft registration form  as well as Stripe payment (deposit) so again very unlikely any genuine spam would get through. However, due to clientele names and emails, sometimes these submissions do get pulled out as spam and it affects the automated workflow.

It would be ideal, if we could disable spam on a per form basis, so all entries for some forms come through regardless. 

Thanks
Terry

**Comment by kjmartens** [comment]
Thanks for your feedback @terryupton. 🙂

**Comment by kjmartens** [comment]
This will be available in the next major version of Freeform. 🙂

**Reply by terryupton** [reply]
Hey @kjmartens will this be coming to Freeform 3 and 4? 
Any idea on when this might be available, as client has asked to follow-up.
Thanks :-)

**Reply by kjmartens** [reply]
Unfortunately this will likely not be carried back to v3 and v4. Freeform 5 is a massive update and overhaul, thus allowing us to easily address feature additions like this as we work with the new architecture and improved interface.

We still might be able to squeeze in an option for this in v4 at least, if it's very necessary.

**Reply by terryupton** [reply]
Is Freeform 5 intended for Craft 5?

**Reply by kjmartens** [reply]
We liked the idea of having Freeform's versions match Craft's, but our timeline progressed a lot faster and Freeform 5 is scheduled for release this summer (and is being built on Craft 4). Depending on Craft's actual release schedule for v5 (and how different it is from v4), we'll see how that plays into things. At any rate, it will be available on Craft 4. 🙂

**Comment by kjmartens** [comment]
This is now available in Freeform 5 (currently in beta).
